setup.cfg

```diff
@@ -12,15 +12,15 @@ max-line-length = 140
 # flake8 is a combination of pyflakes & pycodestyle.
 # Unfortunately we have to mostly duplicate the above, since some tools use
 # pycodestyle's config (eg autopep8) so we can't just define everything under [flake8].
-exclude = .git,__pycache__,.vagrant,node_modules
+exclude = .git,__pycache__,.vagrant,node_modules,tests/jenkins/
 # The ignore list for pycodestyle above, plus our own PyFlakes additions:
 # F403: 'from module import *' used; unable to detect undefined names
 # F405: name may be undefined, or defined from star imports: module
 ignore = E121,E123,E126,E129,E226,E24,E704,W503,E501,F403,F405
 max-line-length = 140
 
 [isort]
-skip = .git,__pycache__,.vagrant,node_modules,migrations
+skip = .git,__pycache__,.vagrant,node_modules,migrations,tests/jenkins/
```

Comment:
It appears that the trailing `/` causes isort to not skip this directory. Probably to be consistent let's remove this from all appearances of this directory in the file.